modules/ols-0-2-0-release-notes.adoc

```diff
@@ -14,6 +14,8 @@ The following enhancements have been made with {ols-offical} 0.2.0:
 
 * This release introduces the {ols-offical} Operator as a Technology Preview release. Previously, the {ols-short} Operator was a Developer Preview release. 
 
+* Existing Developer Preview installations of {ols-offical} version 0.1.6 and earlier that use the preview {olm} (OLM) channel cannot update to the 0.2.0 Technology Preview version. You must switch the update channel to *alpha* and then install the 0.2.0 Technology Preview version.
```

Comment:
Existing Developer Preview installations of {ols-offical} version 0.1.6 and earlier that use the preview {olm} (OLM) channel must switch the update channel to *alpha* and then install the 0.2.0 Technology Preview version. 
@rh-tokeefe  I would move to one single line like above, look better

Reply:
Thanks for the suggestion. Previously I thought the main point we are trying to convey is that there is no direct upgrade path. I like this change. Done.